src/NanopassSharp/Transformations/AddTransformations.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NanopassSharp.Builders;
+
+namespace NanopassSharp.Transformations;
+
+/// <summary>
+/// A transformation which adds a child to a node.
+/// </summary>
+public sealed class AddNodeTransformation : BaseBuilderTransformation
```

Comment:
I feel like most transformations could have been a single lambda. Reminds me of writing an enumerator instead of writing generator functions. Like this works, but it's a lot of crud.